changelog/unreleased/enhancement-inject-customizations

```diff
@@ -0,0 +1,13 @@
+Enhancement: Inject customizations
+
+We have added the possibility to include own header scripts and styles.
+This function is currently still experimental and there is a possibility that the api will change.
+
+For the reasons mentioned, the functionality is not yet documented in the official documentation, but can be used as follows:
+
+* to inject custom css add the following property to your `config.json`, `"styles": [{ "href": "css/custom.css", }]`.
+* to inject custom scripts add the following property to your `config.json`, `"scripts": [{ "src": "css/custom.js", "async": true, }]`.
```

Comment:
```suggestion
* to inject custom scripts add the following property to your `config.json`, `"scripts": [{ "src": "js/custom.js", "async": true, }]`.
```

Reply:
(please also adjust in the PR description)

Reply:
thx, i need glasses or maybe a new 🧠